draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -622,16 +622,6 @@ Security considerations in this section help achieve the following properties:
 * Claim values issued by the Issuer cannot be obtained by the adversary, if the Holder did not intend to disclose them.
 * Claim values that are not issued by the Issuer cannot be successfully verified by the Verifier, if the Holder forged the Disclosures.
 
-## Mandatory digest computation of the revealed claim values by the Verifier
```

Comment:
removing since this has been accidentally re-added in this commit: https://github.com/oauth-wg/oauth-selective-disclosure-jwt/pull/228/commits/183afe5630eaac329f336b615a9ed6049a5ab5de